var/spack/repos/builtin/packages/py-pyeda/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPyeda(PythonPackage):
+    """PyEDA is a Python library for electronic design automation."""
+
+    homepage = "https://github.com/cjdrake/pyeda"
+    url      = "https://pypi.io//packages/source/p/pyeda/pyeda-0.28.0.tar.gz"
+
+    version('0.28.0', sha256='07185f458d5d0b2ba5058da8b95dad6ab7684ceaf41237a25bcd3f005490f59d')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('python@3.3:', type=('build','run'))
```

Comment:
```suggestion
    depends_on('python@3.3:', type=('build', 'run'))
```
This will fix the failing flake8 test.

Reply:
Ping @archxlith

Reply:
Ping ping @archxlith